### DISCUSSION TITLE
On premise deployment of hyperswitch

### DISCUSSION THREAD

**Discussion by jagan-jaya** [juspay/hyperswitch#3880] [General]
Creating this thread to try out the installation of Hyperswitch services in an on-premise cluster with [helm charts](https://github.com/juspay/hyperswitch-helm/tree/main/charts/incubator/hyperswitch-helm). This will help in understanding the difficulties in using the helm-charts and to understand the best practices for an on-prem setup.

**Comment by shaun-forgie** [comment]


**Reply by jagan-jaya** [reply]
hi @shaun-forgie 
This thread is more about k8s/helm installation of hyperswitch using our [helm charts](https://github.com/juspay/hyperswitch-helm/tree/main/charts/incubator/hyperswitch-helm) in an on-premise cluster. We are trying to understand the difficulties the users are facing in an on-premise k8s cluster. Also wanted to understand more on the requirements/standards we need to maintain for an on-premise cluster in-terms of security and reliability so that it is production-ready.
